packages/core-sdk/src/accounts/handler.ts

```diff
@@ -26,25 +32,52 @@ import {
   DisputeResolutionFee,
   DisputeResolverUpdates,
   OptInToSellerUpdateArgs,
-  OptInToDisputeResolverUpdateArgs
+  OptInToDisputeResolverUpdateArgs,
+  CreateCollectionArgs
 } from "./types";
 import { storeMetadataOnTheGraph } from "../offers/storage";
 
+export const INITIAL_COLLECTION_ID = "initial";
+
+export async function findCollectionSalt(args: {
+  sellerToCreate: CreateSellerArgs;
+  contractAddress: string;
+  web3Lib: Web3LibAdapter;
+}): Promise<string> {
+  // collectionSalt is added to the seller admin address to give the sellerSalt that is used to compute the voucher contract address
+  // The seller creation will fail in case sellerSalt already exists
+  // This may happen if the admin address was already assigned to another seller in the past
+  // In that case, we add a random suffix on the collectionId to generate a different salt
+  let collectionSalt = formatBytes32String(INITIAL_COLLECTION_ID);
+  const isAvailable = await isSellerSaltAvailable({
+    adminAddress: args.sellerToCreate.admin,
+    salt: collectionSalt,
+    contractAddress: args.contractAddress,
+    web3Lib: args.web3Lib
+  });
+  if (!isAvailable) {
+    const uuid = Math.floor(Math.random() * 100000000).toFixed(0);
```

Comment:
```suggestion
    const uuid = crypto.randomUUID();
```

Reply:
done

Reply:
I've finally reverted as the crypto symbol is not resolved in tests...